.github/scripts/dapr_bot.js

```diff
@@ -0,0 +1,185 @@
+// list of owner who can control dapr-bot workflow
+// TODO: Read owners from OWNERS file.
```

Comment:
Since this is JS code, do you think this TODO could actually be made?

Reply:
(can be a separate PR :) )

Reply:
Thanks @ItalyPaleAle, created https://github.com/dapr/dapr/issues/5405